File: C++/ej2.cpp
#include <iostream>

int main() {
  std::cin.tie(nullptr);
  std::ios_base::sync_with_stdio(false);

  // TODO: fixme.
  //  Leer el numero final de la sumatoria
  int numeroFinal;
  std::cin >> numeroFinal;

  //  Inicializar la variable para almacenar la sumatoria
  int sumatoria = 0;

  //  Calcular la sumatoria
  for(int i = 1; i <= numeroFinal; ++i){
    sumatoria += i;
  }

  //  Imprimir el resultado
  std::cout << sumatoria << std::endl;


  return 0;
}

File: JAVA/Ej2.java
import java.util.Scanner;
/**
 *  Entrada: 8      Salida: 36          La sumatoria de 1+2+3+4+5+6+7+8 = 36
 * @author HP - MSS | SumatoriaJuanito
 */
public class Ej2 {
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Leer el número final de la sumatoria
        int numeroFinal = scanner.nextInt();

        // Inicializar la variable para almacenar la sumatoria
        int sumatoria = 0;

        // Calcular la sumatoria
        for (int i = 1; i <= numeroFinal; ++i) {
            sumatoria += i;
        }

        // Imprimir el resultado
        System.out.println(sumatoria);

        scanner.close();
    }
    
    
    
}